src/config/Config.ts

```diff
@@ -127,7 +127,9 @@ const conf = {
         `api/iter8/namespaces/${namespace}/experiments/${name}`,
       istioPermissions: 'api/istio/permissions',
       jaeger: 'api/jaeger',
-      jaegerTraces: (namespace: string, app: string) => `api/namespaces/${namespace}/apps/${app}/traces`,
+      appTraces: (namespace: string, app: string) => `api/namespaces/${namespace}/apps/${app}/traces`,
+      serviceTraces: (namespace: string, svc: string) => `api/namespaces/${namespace}/services/${svc}/traces`,
+      workloadTraces: (namespace: string, wkd: string) => `api/namespaces/${namespace}/workloads/${wkd}/traces`,
```

Comment:
I think this is not yet in the backend, isn't it ?

Reply:
yes I'm a bit ahead of time for this one ... first i thought i would do both tasks at the same time..
I can remove if you prefer, but anyway, I'll start to work on the workload case right after that one